app/graph/types/project_type.rb

```diff
@@ -42,6 +42,12 @@
     }
   end
 
+  connection :smooch_user_fields, -> { DynamicAnnotationFieldType.connection_type } do
```

Comment:
Sawy, it's not a good idea to have GraphQL fields or connections that are specific for some annotation types or field names because it doesn't scale... we can't change the schema every time we want to expose an annotation. Ideally let's try to do something like we already have for project media annotations, e.g., a generic field/connection and we can pass the annotation type / field type as an argument: https://github.com/meedan/check-api/blob/develop/app/graph/mutations/graphql_crud_operations.rb#L367-L375

Reply:
Removed this code and retrieve the slack channel in the requests logs by cache the value during field creating and get the value given request data if cache not exists based on our discussion